src/vs/platform/terminal/node/ptyHostService.ts

```diff
@@ -104,7 +105,8 @@ export class PtyHostService extends Disposable implements IPtyService {
 					VSCODE_LAST_PTY_ID: lastPtyId,
 					VSCODE_AMD_ENTRYPOINT: 'vs/platform/terminal/node/ptyHostMain',
 					VSCODE_PIPE_LOGGING: 'true',
-					VSCODE_VERBOSE_LOGGING: 'true' // transmit console logs from server to client
+					VSCODE_VERBOSE_LOGGING: 'true', // transmit console logs from server to client,
+					VSCODE_RECONNECT_CONSTANTS: JSON.stringify(this._reconnectConstants)
```

Comment:
Probably better to pass through the numbers directly since they're going in the env:

```
VSCODE_RECONNECT_GRACETIME: ...,
...
```